app/models/concerns/host_common.rb

```diff
@@ -130,11 +130,11 @@ def crypt_root_pass
   end
 
   def param_true?(name)
-    params.has_key?(name) && LookupKey::TRUE_VALUES.include?(params[name])
+    params.has_key?(name) && Foreman::Cast.to_bool(value)
   end
 
   def param_false?(name)
-    params.has_key?(name) && LookupKey::FALSE_VALUES.include?(params[name])
+    params.has_key?(name) && Foreman::Cast.to_bool(value)
```

Comment:
Check for !nil?, as to_bool returning false will cause this whole method to evaluate to false.